iceoryx_examples/waitset/README.md

```diff
@@ -152,7 +153,7 @@ for (auto i = 0; i < NUMBER_OF_SUBSCRIBERS; ++i)
     auto& subscriber = subscriberVector.back();
 
     subscriber.subscribe();
-    subscriber.attachTo(waitset, iox::popo::SubscriberEvent::HAS_NEW_SAMPLES, 1, subscriberCallback);
+    subscriber.enableEvent(waitset, iox::popo::SubscriberEvent::HAS_SAMPLES, subscriberCallback);
```

Comment:
Ì guess you just did not have the time to update this?

Reply:
readmes are fixed